app/src/main/java/com/example/cheatchess/ChessView.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.cheatchess
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import android.util.AttributeSet
+import android.view.View
+
+class ChessView(context: Context?, attrs: AttributeSet?) : View(context, attrs) {
+    private val paint = Paint()
+    private val rectSide = 130f
+    private val marginLeft = 20f
+    private val marginTop = 200f
+
+    override fun onDraw(canvas: Canvas?) {
+        super.onDraw(canvas)
+        for (i in 0..7)
```

Comment:
I would make this more clear by using x and y.

Reply:
I used row and col